platform/ios/src/MGLMapView.mm

```diff
@@ -1566,7 +1604,11 @@ - (void)handleDoubleTapGesture:(UITapGestureRecognizer *)doubleTap
             [self animateWithDelay:MGLAnimationDuration animations:^
              {
                  [weakSelf unrotateIfNeededForGesture];
+                 [weakSelf notifyScaleBarGestureDidEnd];
              }];
+        } else {
+            [self unrotateIfNeededForGesture];
+            [self notifyScaleBarGestureDidEnd];
```

Comment:
`-[MGLMapView handlePanGesture:]` calls `-notifyGestureDidBegin`.
We have to keep them separated if non-zoom-related gestures like panning shouldn't trigger the scale bar.
Should we extract into `-notifyGestureDidBegin` and `-notifyZoomChangingGestureDidBegin`?
 /cc @friedbunny